src/lib/process_crawl.js

```diff
@@ -1,27 +1,197 @@
 var rc_util = require('rippled-network-crawler/src/lib/utility.js');
+var crawler = require('rippled-network-crawler/src/lib/crawler.js');
```

Comment:
Good point
I'll make the processing fully independent of `rippled-network-crawler`